frontend/renderer/app.ts

```diff
@@ -27,7 +27,17 @@ function createApp(pageContext: PageContext, isClient = true) {
         DefaultApolloClient,
         createApolloClient(pageContext.publicEnv.ENDPOINTS)(getToken, isClient),
       )
-      provide('authService', new AuthService(pageContext.publicEnv.AUTH))
+      try {
+        provide('authService', new AuthService(pageContext.publicEnv.AUTH))
+      } catch {
+        if (navigator.cookieEnabled) {
+          toast.error('🍪 Please enabled cookies!', {
```

Comment:
```suggestion
          toast.error('🍪 Please enable cookies!', {
```

Reply:
It would be nice to translate this string.

Reply:
And maybe it's better to write 
```suggestion
          toast.error('🍪 Please enable cookies in your browser!', {
```